google-cloud-datastore/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.togglz</groupId>
+        <artifactId>togglz-project</artifactId>
+        <version>2.4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>togglz-google-cloud-datastore</artifactId>
```

Comment:
Any idea for a shorter module name? Perhaps omit the `google` part? No sure about that. I'm just sharing my thoughts because the module name looks so long.

BTW: The Google App Engine module is also just called `togglz-appengine`. So perhaps omitting the `google` part would make sense?